cloud-minecraft/cloud-paper/src/main/java/cloud/commandframework/paper/AsyncCommandSuggestionsListener.java

```diff
@@ -36,8 +51,68 @@
 
     private final PaperCommandManager<C> paperCommandManager;
 
+    private final BiConsumer<AsyncTabCompleteEvent, List<Completion>> completionsApplier;
+
+    @SuppressWarnings("ConstantConditions")
     AsyncCommandSuggestionsListener(final @NonNull PaperCommandManager<C> paperCommandManager) {
         this.paperCommandManager = paperCommandManager;
+        if (paperCommandManager.hasCapability(CloudBukkitCapabilities.PAPER_TOOLTIPS)) {
+            BiFunction<String, Message, AsyncTabCompleteEvent.Completion> completionWithDescription;
+            if (Audience.class.isAssignableFrom(Player.class)) { //If we use native adventure
```

Comment:
```suggestion
            if (Audience.class.isAssignableFrom(Player.class)) { // If we use native adventure
```

(also, is this ok for running when Paper doesn't have adventure present? I think the capability check already ensures that this only runs on post-adventure releases)

Reply:
I couldn't find a version of AsyncTabCompleteEvent.Completion without adventure, as before it appeared it was using basic strings, and no support for tooltips

Reply:
so tooltip capability already means that we have adventure